checks.yml

```diff
@@ -16,7 +16,7 @@ linter:
       - semi: error
   code_compilation:
   - name: compile
-    command: npm run build
-  unit_tests:
-  - name: run_tests
-    command: npm test
\ No newline at end of file
+    command: npm run build\n
+  unit_tests:\n
+  - name: run_tests\n
+    command: npm test\n
```

Comment:
```suggestion
    command: npm run build
  unit_tests:
  - name: run_tests
    command: npm test
```

Reply:
The newline to be added is at the end of the file to prevent the red icon from showing.